index.html

```diff
@@ -0,0 +1,153 @@
+                <p class="date font-face">COURSE DURATION IS 10 MONTHS + LIFE TIME SUPPORT</p>
+                <p class="location font-face">We are fully online | Support team available 24 hours |</p>
+            </div>
+        </div>
+    </section>
+
+    <section class="main-program">
+        <div class="program-bg"></div>
+        <div class="p-cont">
+            <p class="program-title">Main Program</p>
+            <hr>
+        </div>
+        
+        <div class="container grid">
+            <div class="card g-col-12 g-col-md-2">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/lecturer.png" alt="lecture image">
+                    <p class="t1">Lecture</p>
+                </div>
+                <p class="program-disc">Watch some of our meterials from various expect across the world.</p>
+            </div>
+            <div class="card g-col-12 g-col-md-2">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/board.png" alt="lecture image">
+                    <p class="t1">CC Exhibition</p>
+                </div>
+                <p class="program-disc">Our board contain timetables and announcements.</p>
+            </div>
+            <div class="card g-col-12 g-col-md-2">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/chat.png" alt="lecture image">
+                    <p class="t1">Forum</p>
+                </div>
+                <p class="program-disc">Have questions, drop us as chat.</p>
+            </div>
+            <div class="card g-col-12 g-col-md-2">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/workshop.png" alt="lecture image">
+                    <p class="t1">Workshop</p>
+                </div>
+                <p class="program-disc">We hold workshops, so no one is left behind.</p>
+            </div>
+            <div class="card g-col-12 g-col-md-2">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/meeting.png" alt="lecture image">
+                    <p class="t1">CC ignite</p>
+                </div>
+                <p class="program-disc">We have weekly meetings to address challenges faced.</p>
+            </div>
+        </div>
+        <div class="button">
+            <a class="start-learning" href="#">Start Learning</a>
+        </div>
+        <p class="whole-program font-face">SEE THE WHOLE PROGRAM</p>
+    </section>
+    
+    <!-- speaker section -->
+    <section class="container main-speakers">
+        <div class="p-cont">
+            <p class="program-title">Featured Speakers</p>
+            <hr>
+        </div>
```

Comment:
- [ ] [REQUIRED] On the mobile homepage, kindly remove the excess space you have between the `Start Learning` button and `Featured Speaker` header so your design can look more professional.
 #### Your Implementation
![Screenshot from 2022-04-21 17-28-33](https://user-images.githubusercontent.com/58692590/164508141-8bc5ef22-bd14-4f65-a5a9-ce82cb375996.png)